recipes/LibriSpeech/ASR/transducer/train.py

```diff
@@ -274,11 +274,12 @@ def on_stage_end(self, stage, stage_loss, epoch):
                 train_stats=self.train_stats,
                 valid_stats=stage_stats,
             )
-            # We save multiple checkpoints as we will average them!
             self.checkpointer.save_and_keep_only(
                 meta={"WER": stage_stats["WER"], "epoch": epoch},
-                min_keys=["WER"],
+                min_keys=["ACC"],
+                num_to_keep=self.hparams.avg_checkpoints,
```

Comment:
we might need to retrain this model to better understand what it means in practice. Does it worsen the results or ?